pages/settings/admin/dlna.vue

```diff
@@ -0,0 +1,27 @@
+<template>
+  <settings-page page-title="settingsSections.admin.dlna.name">
+    <template #content>Placeholder for DLNA settings</template>
```

Comment:
Probably easier just to leave these pages out for now. And add them when they're functional, instead of placeholders

Reply:
The idea of placeholders is so we don't forget about adding the titles of it and the middleware redirection. We already had some of this stuff missing (api keys was missing the middleware and logs activity was missing the ``head`` iirc). Also, without them, clicks on the navigation drawer for the pages will keep multiple components in an active state (as no redirection will be happening), which looks really bad, specially if we're going to do a preview sooner than later.

Reply:
My only thoughts are since we are heading to a RC in the next few weeks, it'd be better not to have blank pages with placeholder text, and only add them when the page is implemented